test/cpp_extensions/open_registration_extension/torch_openreg/README.md

```diff
@@ -125,7 +125,7 @@ graph LR
 
 ### Autoload
```

Comment:
Add a section for `Autoload`:

Take a introduction about background and design of `Autoload`, and add detailed description to show how the new accelerator to use `Autoload` mechinasm